### PR TITLE
[15.0][ADD] `project_forecast_line_bokeh_chart`

### PR DESCRIPTION
**Pull request by yankinmax**
Depends on:
- [ ] https://github.com/OCA/project/pull/957

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]
Would be good to add some unit tests here

**Comment by yankinmax** [comment]
Will add tests later

File: project_forecast_line_bokeh_chart/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module adds forecast line report to project.
```
Comment (gurneyalex) [inline project_forecast_line_bokeh_chart/readme/DESCRIPTION.rst]:
```suggestion
This module improves the reports of project_forecast_line by using the bokeh widget available in OCA/web

```

**Review by gurneyalex — APPROVED** [review]

**Comment by leemannd** [comment]
Hello @yankinmax Could you remove the commit dependency in order to merge it?

**Comment by yankinmax** [comment]
@leemannd done it

**Pull request merged**